frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/TypeSelector/TypeSelector.tsx

```diff
@@ -33,6 +33,7 @@ export const BehaviorOption = ({
     isSelected={selected}
     onClick={onClick}
     disabled={disabled}
+    aria-selected={selected}
```

Comment:
Good call on adding `aria-selected` :+1: 
However I believe we should add this attribute inside of `SidebarItem.Selectable` instead - this way we'd handle all `SidebarItem.Selectable` usages at once.